Makefile

```diff
@@ -403,9 +403,9 @@ KBUILD_CXXFLAGS   := -c -g -std=c++11 \
 
 KBUILD_AFLAGS   := -c -g -xassembler-with-cpp -D_ASMLANGUAGE
 
-LDFLAGS += $(call ld-option,-nostartfiles)
+LDFLAGS += $(if $(filter $(ARCH), mips),,$(call ld-option,-nostartfiles))
 LDFLAGS += $(call ld-option,-nodefaultlibs)
-LDFLAGS += $(call ld-option,-nostdlib)
+LDFLAGS += $(if $(filter $(ARCH), mips),,$(call ld-option,-nostdlib))
```

Comment:
Same as above comment by Sebastian, why do we need this? What symbols are missing without it? We explicitly include libgcc later on in the build IIRC